templates/incoming_ref_missing.j2

```diff
@@ -0,0 +1,36 @@
+Hello,
+we're closing this pull request because the original PR-bound branch
+went missing. (This PR's fork repository or branch has been deleted by
+the owner).
+
+@{{ submitter }}, if you with to restore the PR, please, re-create
+a branch in the same fork repo and with the exactly same name and reopen
+this pull request. Or submit a new one.
```

Comment:
Clarify what "a new one" means. 
A new branch and therefore PR.

Reply:
A new PR. feel free to submit a "suggestion" by pressing Ctrl+G in comment box :)